script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -512,37 +574,40 @@ However, it requires a millennium-class basis in order to make a Hemonomic Princ
 [sha:4fde2c2628b5601d767c29c05646c69e0838cc83]{
 -- Page 612, Offset 18115.
 -- 「そうなんだけど……今にして思うと、アイツ、ちゃんと耐えきっていたみたい。
-"Sure, but... now that I think about it, that guy looked like he was able to handle it properly.
+"Sure, but... now that I think about it, that guy looked like he was able to withstand it pretty well.
 }
 [sha:601c575a014f7a176e961bc5b894fc7a5827c162]{
 -- Page 612, Offset 18116.
 -- ……死徒としてはⅥ階梯クラスだったのに、なんであそこまで自我が残っていたのかしら……」
-Even though he was only Rank VI as a Dead Apostle, I wonder why his ego still remained as much as it did..."
+He was only a Rank VI Dead Apostle, I'm not sure how he was able to retain as much of his ego as he did..."
 }
 [sha:2c4ee014a377237aae50df1e1ee67fcb8ea2285a]{
 -- Page 613, Offset 18117.
 -- 「まあ、とんでもない連中だって事は分かったよ。
-"Well, I understand now that they're a group of outrageous people.
+"Well, I get the picture that they're not a group of people you want to mess with.
 }
 [sha:c742f19f2ea84dd90faa0164b8be48fd60f22642]{
 -- Page 613, Offset 18118.
 -- それが何の間違いか俺たちの街にやってきて、俺たちが始末した。
-He came to our city for whatever reason, and then we took care of it.
+But by some stroke of bad luck, one showed up in our city,
+ and we dealt with them.
 }
 [sha:9a96fe5a0c24fdbd4f82f0d7b2b475b33ab141f2]{
 -- Page 613, Offset 18119.
 -- ……あまり考えたくないんだけど。これって、他の祖を怒らせる事になるのか？」
-I really don't want to beg the question but... would doing this cause the other Ancestors to become angry?"
+I'm not sure I want to hear the answer to this, but...
+ Is that something that will have angered the other Ancestors?"
 }
 [sha:cd8892992b1882580a32640a0ae638f9b67826a9]{
 -- Page 614, Offset 18120.
 -- 「あ、それは大丈夫。二十七祖は基本、敵同士だから。
-"Ah, you don't have to worry about that. The Twenty-Seven Ancestors are fundamentally mutual enemies.
+"Ah, you don't have to worry about that.
+ The Twenty-Seven Ancestors are fundamentally at odds with each other.
 }
 [sha:67914a8456437c261eb5139b077df7839490941c]{
 -- Page 614, Offset 18121.
 -- こんな島国でヴローヴが消えたところで気にする祖はいないわ」
-There isn't a single Ancestor that would care about Vlov disappearing in an island like this."
+There isn't a single Ancestor that would care about Vlov's disappearance in this island nation."
```

Comment:
don't like "care" here because the implication above is that it might benefit them. maybe "lose sleep over", "mind" etc. something that specifically clarifies it doesnt _bother_ them.